aries-site/src/layouts/content/Example/Container.js

```diff
@@ -24,7 +24,8 @@ export const Container = ({
 
   let height;
   if (heightProp) height = heightProp;
-  else if (template || screenContainer) height = aspectHeight;
+  else if (template) height = aspectHeight;
+  else if (screenContainer) height = heightProp || aspectHeight;
```

Comment:
Any reason not to have `else if (screenContainer || template) height = heightProp || aspectHeight;` and allow either presentation to define their own height?

Reply:
soft opinion, i'm fine leaving as you've changed it.

Reply:
Done.